pkg/cosign/verify.go

```diff
@@ -34,6 +33,7 @@ import (
 	v1 "github.com/google/go-containerregistry/pkg/v1"
 	"github.com/pkg/errors"
 
+	ssldsse "github.com/secure-systems-lab/go-securesystemslib/dsse"
```

Comment:
Are we using this consistently everywhere?  I'm guessing "no" because it required changes to `go.mod`.  Want to make sure nothing else should be using this?

Reply:
We're using it in `sigstore/dsse`. The go.mod changes were to update sigstore/sigstore to a version which exposed `dsse.VerifierAdapter`